osu.Game/Screens/Edit/Compose/Components/DistanceSnapGrid.cs

```diff
@@ -82,6 +85,9 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             beatDivisor.BindValueChanged(_ => updateSpacing(), true);
+
+            distanceSpacingMultiplier = SnapProvider.DistanceSpacingMultiplier.GetBoundCopy();
+            distanceSpacingMultiplier.BindValueChanged(_ => updateSpacing(), true);
```

Comment:
`beatDivisor` is already calling `updateSpacing()` once above via the `BindValueChanged(..., true)` call. Probably need just one of those spacing updates.